pandas/core/sorting.py

```diff
@@ -40,7 +40,7 @@
     from pandas import MultiIndex
     from pandas.core.indexes.base import Index
 
-_INT64_MAX = np.iinfo(np.int64).max
+_INT64_MAX = lib.i8max
```

Comment:
can u fix these rather than just renaming

Reply:
alt change the ones in the .pyx to be uppercase 

it's not at all clear that these are actually the same

Reply:
Updated+green